src/features/Main/sections/Hero/index.tsx

```diff
@@ -1,6 +1,28 @@
+import Image from 'next/image';
+import HeroImage from '../../../../../public/assets/img/Hero.svg';
 import { FC } from 'react';
 
+
 const Hero: FC = () => {
-  return <div>Hero</div>;
+  return (
+    <div className="container mx-auto px-4 flex flex-col md:flex-row items-center justify-between my-6">
+      <div className=" md:w-2/5 flex flex-col gap-6">
+        <h1 className="text-5xl font-semibold text-[#0F3665]">
```

Comment:
@FarhanSaleh I think the header of text should be minimized a little more when responsive

Reply:
ok i'll try it

Reply:
@FarhanSaleh next when update the ui, u should update the description on the pull request again, lets merge